src/Steps/Release/UpdateTranslations.php

```diff
@@ -221,6 +239,31 @@ protected function pullSource(OutputInterface $output, $modules)
     }
 
     /**
+     * Tidy yaml files using yamlclean ruby gem
+     *
+     * @param OutputInterface $output
+     * @param Module[] $modules List of modules
+     */
+    protected function cleanYaml(OutputInterface $output, $modules)
+    {
+        foreach ($modules as $module) {
+            $name = $module->getName();
+            $this->log(
+                $output,
+                "Cleaning YAML sources for <info>{$name}</info>"
+            );
+
+            foreach (glob($module->getLangDirectory()."/*.yml") as $sourceFile) {
+                $cleanCommand = sprintf(
+                    '(yamlclean %1$s > %1$s.new && mv %1$s.new %1$s)',
+                    $sourceFile
```

Comment:
Writing to a new file and then renaming because I couldn’t get it to write to the same file immediately.

```
$ yamlclean foo.yml > foo.yml  # Results in the file containing just "---false"
$ yamlclean foo.yml >> foo.yml # Appends YAML from stdout as expected
```

Perhaps something to do with the file already being open? Again, not sure.